p2p/src/network/scheduler_effectful/p2p_network_scheduler_effectful_effects.rs

```diff
@@ -0,0 +1,82 @@
+use redux::ActionMeta;
+use std::net::SocketAddr;
+
+use crate::{MioCmd, P2pCryptoService, P2pMioService};
+
+use super::{super::*, *};
+
+impl P2pNetworkSchedulerEffectfulAction {
+    pub fn effects<Store, S>(self, _meta: &ActionMeta, store: &mut Store)
+    where
+        Store: crate::P2pStore<S>,
+        Store::Service: P2pMioService + P2pCryptoService,
+    {
+        match self {
+            P2pNetworkSchedulerEffectfulAction::InterfaceDetected { ip, port } => {
```

Comment:
Now that I'm noting this, do we want to listen on any detected interface? Do we really want this behavior? @vlad9486

Reply:
@dkuehr May be useful on mobile devices where cellular and wifi are turned on/off during runtime.